ios/sdk/src/MPVolumeViewManager.m

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright @ 2017-present Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <React/RCTUIManager.h>
+#import <React/RCTViewManager.h>
+
+@import MediaPlayer;
+
+
+@interface MPVolumeViewManager : RCTViewManager
+@end
+
+@implementation MPVolumeViewManager
+
+RCT_EXPORT_MODULE()
+
+- (UIView *)view {
+    MPVolumeView *volumeView = [[MPVolumeView alloc] init];
+    volumeView.showsRouteButton = YES;
+    volumeView.showsVolumeSlider = NO;
+
+    return (UIView *) volumeView;
+}
+
+RCT_EXPORT_METHOD(show:(nonnull NSNumber *)reactTag) {
+    [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry) {
```

Comment:
Can you please attempt to come up with formatting which results in a maximum of 80 characters per line and looks somewhat consistent with the existing source code?

Reply:
I tried but I couldn't get it < 80 and it kinda hurt readability. I'll give it another go.